src/platform/plugins/shared/unified_histogram/public/__mocks__/suggestions.ts

```diff
@@ -185,12 +185,12 @@ export const histogramESQLSuggestionMock = {
       '662552df-2cdc-4539-bf3b-73b9f827252c': {
         index: 'e3465e67bdeced2befff9f9dca7ecf9c48504cad68a10efd881f4c7dd5ade28a',
         query: {
-          esql: 'from kibana_sample_data_logs | limit 10 | EVAL timestamp=DATE_TRUNC(30 second, @timestamp) | stats results = count(*) by timestamp | rename timestamp as `@timestamp every 30 second`',
+          esql: 'from kibana_sample_data_logs | limit 10 | EVAL timestamp=DATE_TRUNC(30 second, @timestamp) | stats results = count(*) by timestamp',
```

Comment:
Is it correct that when adding to a dashboard, the chart does not show the same long time range?

![Jan-27-2025 14-07-17](https://github.com/user-attachments/assets/11856814-def7-4063-9312-efc3c867a8d7)

Reply:
This is happening also in main Julia. It is due to an ES limitation.

Reply:
Hm, how do we have a different view on Discover page then?

Reply:
It seems so, not exactly sure why it looks differently in Discover though. I was referring to this btw https://github.com/elastic/elasticsearch/issues/111483

Reply:
I think is because of this Jul https://github.com/elastic/kibana/blob/main/src/platform/plugins/shared/unified_histogram/public/chart/hooks/use_lens_props.ts#L91

Reply:
Since dashboard knows the current time range then it could pass it to Lens and it would be fixed on Dashboard page too even without ES support, right?

Reply:
I think the viz team wants to solve it when we have this https://github.com/elastic/elasticsearch/issues/111483 from ES side. Right @markov00 ?

Reply:
That's the idea, but this doesn't blocks us from specifying directly the timerange in the chart domain until this feature is available in ESQL

Reply:
yes I agree, we should.